index.html

```diff
@@ -0,0 +1,215 @@
+              <li class="nav-item">
+                <a href="#" class="nav-link campaign">CC Campaign</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- HEADER ENDS -->
+
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-page container">
+        <p class="greet">"Lead the world through innovations."</p>
+        <h1>
+          Connecting Minds, <br />
+          Empowering Change <br />
+          2023
+        </h1>
+        <p class="description">
+          "The Innovative Leadership Conference is a premier gathering of
+          professionals and thought leaders, exploring the latest trends and
+          innovations. Join us to network, share knowledge and advance your
+          career."
+        </p>
+        <time datetime="2023-05-20T19:30">May 20, 2023 @ 3:00pm</time>
+        <p class="location">
+          Kwame Nkrumah University of Science and Technology (KNUST), Kumasi.
+          Ghana.
+        </p>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- PROGRAMS STARTS -->
+    <section class="program">
+      <div class="container">
+        <h2>Main Program</h2>
+        <hr />
+
+        <div class="program-contents">
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-analysis"></i>
+              <h3>Trends</h3>
+            </div>
+            <p>Level up your brand with the latest digital marketing trends.</p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Brands</h3>
+            </div>
+            <p>Branding strategies in the age of the Metaverse</p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Forum</h3>
+            </div>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Works</h3>
+            </div>
+            <p>
+              Try creating your own work using open source license rather than
+              just watching.
+            </p>
+          </div>
+
+          <div class="content">
+            <div class="icon-title">
+              <i class="uil uil-palette"></i>
+              <h3>Design</h3>
+            </div>
+            <p>
+              Get the opportunity to connect with different brands to showcase
+              your creative prowess.
+            </p>
+          </div>
```

Comment:
- [ ] Good job, @kessie2862 :clap: However, I suppose these texts can be aligned centrally unlike the `left-alignment` like we have in the template to really make the design more appealing :wink: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/233465394-6cf880e7-7447-404a-936d-e4c248a72b4f.png)|![image](https://user-images.githubusercontent.com/99715640/233465562-8b90d792-b825-4c51-872d-87491a6c297a.png)|